packages/analytics-js/src/components/core/Analytics.ts

```diff
@@ -273,7 +277,8 @@ class Analytics implements IAnalytics {
    */
   onPluginsReady() {
     this.errorHandler.init(this.externalSrcLoader);
-
+    // Set plugins loaded flag to true
+    state.reporting.isErrorReportingPluginLoaded.value = true;
```

Comment:
this is not accurate here. This is all plugins loaded not the particular one.

It is cleaner to do that in the plugin init method itself taht is invoked as soon as the plugin is loaded